python/ray/llm/_internal/serve/config_generator/utils/constants.py

```
@@ -0,0 +1,17 @@
+import os
+from ray.llm._internal.serve.configs.constants import RAYLLM_HOME_DIR
+
+TEMPLATE_DIR = os.path.normpath(
+    os.path.join(
+        os.path.dirname(os.path.abspath(__file__)), "..", "base_configs", "templates"
+    )
+)
+
+HF_TOKEN_FILE = os.path.join(RAYLLM_HOME_DIR, ".HF_TOKEN")
+
+DEFAULT_DEPLOYMENT_CONFIGS_FILE = "default_deployment_configs.yaml"
+
+
+def get_lora_storage_uri():
+    artifact_storage = os.environ.get("RAYLLM_LORA_STORAGE_URI", "s3://my-bucket")
```

nit: I feel we shouldn't give a fake default like `s3://my-bucket` which will only fail when users are running service and actually sending a lora request. We should probably log a warning or make some assumptions to support a local lora directory as the default.

---

sounds good. supporting lora path actually sounds more practical. I will make it a local path for now. We need to follow up with that local path dynamic lora path support anyways.